src/main/scala/rise/core/Expr.scala

```diff
@@ -1,46 +1,82 @@
 package rise.core
 
+import semantics._
 import rise.core.types._
 import rise.core.DSL.Type.TypeEqual
 import rise.core.ShowRise._
 
+object alphaEquiv {
+  type TYPEEQ = Type => Type => Boolean
```

Comment:
maybe `TypeEq`?

Reply:
Yeah, sounds better. Btw I'm thinking about moving all the equality stuff to `core/Equality` and `core/Types/Equality`, how does this sound?

Reply:
I think `core.equality` sounds good to me and I would include the type equality there as well (there is not a lot of code, right?)

Reply:
No, there isn't, sounds good!